File: src/Jering.VectorArtKit.WebApi/ResponseModels/Account/ResetPasswordResponseModel.cs
using System;
using Jering.VectorArtKit.WebApi.ResponseModels.Shared;
using Microsoft.AspNetCore.Mvc;
using Jering.Mvc;

namespace Jering.VectorArtKit.WebApi.ResponseModels.Account
{
    public class ResetPasswordResponseModel: IErrorResponseModel
    {
        public bool AuthenticationError { get; set; }
        public bool AntiForgeryError { get; set; }
        public SerializableError ModelState { get; set; }
        public bool InvalidToken { get; set; }
        public bool InvalidEmail { get; set; }

        public bool ExpectedError { get; set; }

        public string ErrorMessage { get; set; }
    }
}

File: src/jering-vectorartkit-webclient/src/api/response-models/reset-password.response-model.ts



    export interface ResetPasswordResponseModel {
	    authenticationError?: boolean;
	    antiForgeryError?: boolean;
	    modelState?: { [key: string]: any; };
	    invalidToken?: boolean;
	    invalidEmail?: boolean;
	    expectedError?: boolean;
	    errorMessage?: string;
    }

    

        

